chromium_src/chrome/browser/metrics/desktop_session_duration/desktop_profile_session_durations_service.h

```diff
@@ -0,0 +1,16 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_METRICS_DESKTOP_SESSION_DURATION_DESKTOP_PROFILE_SESSION_DURATIONS_SERVICE_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_METRICS_DESKTOP_SESSION_DURATION_DESKTOP_PROFILE_SESSION_DURATIONS_SERVICE_H_
+
+#include "brave/components/sync/driver/brave_sync_session_durations_metrics_recorder.h"
+
+#define SyncSessionDurationsMetricsRecorder \
+  BraveSyncSessionDurationsMetricsRecorder
```

Comment:
This will cause 
```
#define SyncSessionDurationsMetricsRecorder BraveSyncSessionDurationsMetricsRecorder
#define SyncSessionDurationsMetricsRecorder BraveSyncSessionDurationsMetricsRecorder
#undef SyncSessionDurationsMetricsRecorder
#include "../../../../../../chrome/browser/metrics/desktop_session_duration/desktop_profile_session_durations_service.cc"
#undef SyncSessionDurationsMetricsRecorder
```
when its chromium override cc include it. Then .cc ends up replacing nothing. cc @bridiver

Reply:
However, https://github.com/brave/brave-core/pull/5616/files#diff-669e15eca2ac590eb9cd6c1e740740bbR6 extra header guard prevents it from happening.